Gateway/Request/RefundDataBuilder.php

```diff
@@ -82,10 +89,10 @@ public function build(array $buildSubject): array
         // check if it contains a partial payment
         $orderPaymentCollection = $this->orderPaymentCollectionFactory
             ->create()
-            ->addFieldToFilter('payment_id', $payment->getId());
+            ->addFieldToFilter(OrderPaymentInterface::PAYMENT_ID, $payment->getId());
 
         // partial refund if multiple payments check refund strategy
-        if ($orderPaymentCollection->getSize() > self::REFUND_STRATEGY_ASCENDING_ORDER) {
+        if ($orderPaymentCollection->getSize() > 1) {
```

Comment:
Why is this changing?

Reply:
This condition checks the size of the collection and it is not relevant to the constant. It's literally an integer comparison. `REFUND_STRATEGY_ASCENDING_ORDER` is used for the values of the dropdown refund strategy configuration field (meaning refund from last payment first).